department-of-veterans-affairs/abd-vro
# Mcp 1867 - Remove logging credentials and restore some tests.

Removing logging some credential information for security concern and restore some tests.

1. Removing some credential information logging for security concerns, and minor code refactor for lint complaints.
2. Restoring some commented out unit tests for VroController as mentioned in MCP-1875.


### What was the problem?

Some credentials were shown in the log, and some unit tests were failing.

### How does this fix it?

Remove related logs, and fix the unit tests.

### Jira Tickets

MCP-1867, MCP-1875.

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How to test this PR

- Step 1: Build the application.
- Step 2: Check the test result for app/VroController. The successful test result should be seen (previous code should not produce all the rest results seen).

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
